htdocs/core/class/commonobject.class.php

```diff
@@ -5602,7 +5603,9 @@ public function call_trigger($triggerName, $user)
 	{
 		// phpcs:enable
 		global $langs, $conf;
-
+		if (!empty(self::TRIGGER_PREFIX) && strpos($triggerName, self::TRIGGER_PREFIX . '_') !== 0) {
+			throw new Exception('The trigger "' . $triggerName . '" does not start with "' . self::TRIGGER_PREFIX . '_" as required.');
```

Comment:
What happen when you do a "throw" outside a try catch context ?

Reply:
In a standard apache context, I think this causes a fatal error.

Reply:
I changed it accordingly.